packages/dates/src/base-date-picker/BaseDatePicker.tsx

```diff
@@ -3,20 +3,29 @@ import {
   IconButton,
   InputAdornment,
   InputProps,
-  Popover,
+  Popover as MuiPopover,
   SvgIcon,
   TextField,
 } from '@material-ui/core';
 import { Clear } from '@material-ui/icons';
 import { mdiCalendarMonth } from '@mdi/js';
 import { useValueObserver } from '@superdispatch/hooks';
-import { mergeRefs, useUID } from '@superdispatch/ui';
+import { Color, mergeRefs, useUID } from '@superdispatch/ui';
 import { forwardRef, Ref, useImperativeHandle, useRef, useState } from 'react';
+import styled from 'styled-components';
 
 export interface InternalBaseDateFieldAPI {
   close: () => void;
 }
 
+const Popover = styled(MuiPopover)`
+  .MuiPaper-rounded {
+    border: 1px solid ${Color.Silver400};
+    box-shadow: 0px 0px 4px 0px rgba(0, 0, 0, 0.05),
+      0px 4px 8px 0px rgba(0, 0, 0, 0.12);
+  }
+`;
+
```

Comment:
@umrzoq-toshkentov Do we need it only on date pickers or on the calendar itself too? Can you clarify with the designer?

Reply:
It seems to be more related DatePicker, but I will ask from designers again in channel

Reply:
@umrzoq-toshkentov You can merge this PR. And if we need this change in the calendar too - you can create a new one.